### ISSUE TITLE
Don't rename mednafen_psx_hw_libretro.so to mednafen_psx_libretro.so

### ISSUE THREAD

**Issue by crocket** [stefan-gr/abendbrot#105]
`games-emulation/libretro-info-1.0_pre20171127` contains `mednafen_psx_hw_libretro.info`.

**Comment by stefan-gr** [comment]
Yes, that's what I was talking about in https://github.com/stefan-gr/abendbrot/issues/104#issuecomment-374192767 
Do you miss core options? The info file has just a different display name and as far as I know, cores provide the options themselves.

**Comment by crocket** [comment]
> make it so that hw and sw cores are install-able at the same time.

How?

**Comment by stefan-gr** [comment]
USE flags and a conditional src_compile phase. It requires a little tinkering similar to bsnes-libretro. Let me slap something up with bsnes as a base later and you can test this with beetle-psx-libretro-9999 in the evening or something around that time. If it works out I will apply the changes to the old versions and rename beetle ebuild to mednafen.

**Comment by stefan-gr** [comment]
It's untested, it should install two cores if software and at least opengl or vulkan are in USE.

**Comment by crocket** [comment]
I built mednafen_psx_libretro.so and mednafen_psx_hw_libretro.so with beetle-psx-libretro.
Both cores run Mega Man X4, but the opengl core has some graphical glitches.

I don't know whether some core options can fix glitches, but they work.

**Comment by stefan-gr** [comment]
> Does mednafen_psx_hw_libretro.so handle both opengl and vulcan?

According to the Makefile both seem to be supported at the same time. See:
https://github.com/libretro/beetle-psx-libretro/blob/master/Makefile#L56
